technology/cassandra/src/main/java/com/github/mizool/technology/cassandra/ClusterProducer.java

```diff
@@ -33,6 +36,12 @@
 class ClusterProducer
 {
     private static final String CASSANDRA_CONTACT_POINTS_PROPERTY_NAME = "cassandra.contactpoints";
+    private static final String
+        CASSANDRA_MAX_REQUESTS_PER_CONNECTION_LOCAL_PROPERTY_NAME
+        = "cassandra.max.requests.per.connection.local";
```

Comment:
when using the period, it must separate logical hierarchy levels. change this to `cassandra.maxRequestsPerConnection.local` and adapt the other name accordingly.

Note: yes, using camel case here is inconsistent with `cassandra.contactpoints`, but that's not worth breaking the API, and restricting to lower case feels too ugly.